endpoint.go

```diff
@@ -612,6 +633,14 @@ func (ep *endpoint) setupDNS() error {
 	return ep.updateDNS(resolvConf)
 }
 
+// EndpointOptionPrimary function returns an option setter for making the
+// endpoint a primary endpoint
+func EndpointOptionPrimary() EndpointOption {
```

Comment:
In that case, the independent `dnet` tool cannot be used as an ingredient as this ties the set-gateway functionality to `docker run`.